superpixel_classification/SuperpixelClassification/SuperpixelClassification.xml

```diff
@@ -138,6 +138,13 @@
       <description>Labels for random input</description>
       <default>Tissue,Background,Marker</default>
     </string-vector>
+    <string-vector>
+      <name>exclude</name>
+      <longflag>exclude</longflag>
+      <label>Exclude Labels</label>
+      <description>Labels excluded from training.  Any feature with this label is treated as unlabelled for the purposes of training</description>
```

Comment:
I think as a separate PR, we need to ensure all labeled superpixels are after all non-labelled superpixels in the certainty metric, since one of the points of this is to avoid asking again on those samples.

Reply:
Sounds good.

Reply:
Looking at the code, maybe excluding these labels should be done on the ALBench side of things.  We currently call `compCertainty.set_batchbald_excluded_samples(labeled_samples)`, but we really want to also deprioritize those labeled samples in the general certainty calculations.  They need to be excluded from batchbald, but deprioritized in other certainty metrics.  If needed, we can discuss that elsewhere -- I'll merge this PR.